README.md

```diff
@@ -148,11 +134,18 @@ data-diff --conf datadiff.toml \
 + 1, returned
 ```
 
+5. Review the output
+
+After running your data-diff job, review the output to identify and analyze differences in your data.
+
 Check out [documentation](https://docs.datafold.com/reference/open_source/cli) for the full command reference.
 
+# data-diff OSS & Datafold Cloud
```

Comment:
Let's move this section right above the contributions section. This doesn't narratively make sense to inject here.

Reply:
fixed